community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -571,6 +574,15 @@ def setup_compute():
 
     log.info("Done setting up compute")
 
+def setup_flex_db(lkp:util.Lookup) -> None:
+    cmd = "mysql -u root -e"
```

Comment:
Is it `flex_db` or `mig_db`? 
I would rather call it `slurm_gcp_db` since it may have more that one use.

Will it work if CloudSQL is used?

Reply:
Changing name to `slurm_gcp_db` in case more things use it going forward 

> Will it work if CloudSQL is used?

Good question, testing now